src/display/process/allProcs.go

```diff
@@ -273,7 +274,7 @@ func AllProcVisuals(dataChannel chan []*proc.Process,
 						procToKill, err := proc.NewProcess(pidToKill)
 						myPage.ProcTable.CursorColor = selectedStyle
 						if err == nil {
-							err = procToKill.Kill()
+							err = procToKill.SendSignal(syscall.SIGTERM)
```

Comment:
I think this should be `SIGKILL` as that is what [`Kill()`](https://pkg.go.dev/github.com/shirou/gopsutil@v3.21.4+incompatible/process#Process.Kill) sends, to maintain consistency

Reply:
I briefly mentioned in the commit message why I chose SIGTERM over SIGKILL, but it is an irrelevant decision either way as it is only temporary. This will eventually be replaced by the list of signals to choose from. We could revisit this if we choose to have a default signal to send.